en/peripherals/camera.md

```diff
@@ -63,14 +63,15 @@ Camera trigger pins are set in the _QGroundControl_ [Actuators](../config/actuat
 The trigger pins can be set by assigning the `Camera_Trigger` function on any FMU output.
 If using trigger setup that requires two pins (e.g. Seagull MAP2) you can assign to any two outputs.
 
-Note however that once an output has been used for camera triggering, the whole PWM group cannot be used for anything else (you can't use another output in the group for an actuator or motor, say).
+Note however that if a _PWM_ output has been used for camera triggering (such as Seagull MAP2), the whole PWM group cannot be used for anything else (you can't use another output in the group for an actuator, motor, or camera capture, because the timer has already been used).
 
 :::note
 At time of writing triggering only works on FMU pins:
```

Comment:
Is this still true? Why? No timer on IO pins?

Reply:
Yes, it's just not implemented.